lib/src/meta.dart

```diff
@@ -126,3 +126,69 @@ void geomEach(dynamic geoJSON, GeomEachCallback callback) {
     featureIndex++;
   }
 }
+
+/// Callback for propEach
+typedef PropEachCallback = dynamic Function(
+    Map<String, dynamic> currentProperties, num featureIndex);
+
+/// Iterate over properties in any [geoJSON] object, calling [callback] on each
+/// iteration. Similar to Array.forEach()
+///
+/// For example:
+///
+/// ```dart
+/// FeatureCollection featureCollection = FeatureCollection(
+///   features: [
+///     point1,
+///     point2,
+///     point3,
+///   ],
+/// );
+/// propEach(featureCollection, (currentProperties, featureIndex) {
+///   someOperationOnEachProperty(currentProperties);
+/// });
+/// ```
+void propEach(dynamic geoJSON, PropEachCallback callback) {
```

Comment:
Couldn't we add a more specific type specification, like `GeoJSONObject`?

---
Because my GeoJSON implementation follows the RFC so strictly, we can use the abstract classes: `GeoJSONObject`, as well as `Geometry` and `GeometryType`.

Reply:
See #12

Reply:
#29 has at least one way to accomplish this, which could be replicated here if we think that is an acceptable way to strongly type these functions